frontend/src/component/changeRequest/ChangeRequest/Changes/Change/FeatureChange.tsx

```diff
@@ -134,7 +134,14 @@ export const FeatureChange: FC<{
                 }
             />
 
-            <Box sx={(theme) => ({ padding: theme.spacing(3) })}>
+            <Box
+                sx={(theme) => ({
+                    padding: theme.spacing(3),
+                    '&:has(.delete-strategy)': {
+                        backgroundColor: theme.palette.error.light,
+                    },
+                })}
+            >
```

Comment:
Because the container is made the same way regardless of what content it will have, we need some way to check whether it is a box for deleting strategies. Luckily, the `has` selector will do that for us. In unsupported browsers, it'll just fall back to using the regular (theme) background color.

Reply:
I think this is fine, its a good compromise

Reply:
Oh, and for the record, caniuse reports that [91.96% of browsers support `:has`](https://caniuse.com/?search=%3Ahas).